admin/admin-model.php

```diff
@@ -70,6 +70,8 @@ public function add_language( $args ) {
 
 		$this->clean_languages_cache(); // Again to set add mo_id in the cached languages list
 		flush_rewrite_rules(); // Refresh rewrite rules
+		$this->get_languages_list(); // Refresh languages.
```

Comment:
```suggestion
		// Refresh languages.
		$this->clean_languages_cache();
		$this->get_languages_list();
		
		flush_rewrite_rules(); // Refresh rewrite rules.
```
- I propose to change the order of the calls as it makes more sense to deal withthe languages all at conce.
- Removed the `Again...| comment. It seems to me that now the first call to `clean_languages_cache()` higher in the method could be removed.
- Maybe we should do the same for `update_language()` for consistency.

Reply:
Done in b173e86.
But I didn't remove the previous call to `clean_languages_cache()` in `add_language()` because otherwise the later `$this->get_language( $args['slug'] )` returns `false`.